src/nrniv/CMakeLists.txt

```diff
@@ -322,3 +322,55 @@ endif()
 # ~~~
 install(TARGETS nrniv nocmodl modlunit DESTINATION bin)
 install(TARGETS nrniv_lib DESTINATION ${NRN_INSTALL_SHARE_LIB_DIR})
+if(MINGW)
+  set(mswindir ${PROJECT_SOURCE_DIR}/src/mswin)
+  add_custom_command(OUTPUT nrnicon.res
+    COMMAND echo "id ICON \"${mswindir}/nrniv10.ico\"" > nrnicon.rc
```

Comment:
There has to be a better way to do this. I believe redirecting stdout is unsafe.

Reply:
check `FILE(WRITE file_path "some text")`